layout.yaml

```diff
@@ -8,4 +8,8 @@ categories:
     icon: fa fa-address-book-o
     articles:
       - article2
-      - article3
\ No newline at end of file
+      - article3
+  - name: Single Entrypoint
```

Comment:
I think this should be `Single-Entrypoint` but I may be wrong. @franzpgarcia ?

Reply:
It's the name of the folder. Maybe I should rename it to `category:`

Reply:
https://cloudops.atlassian.net/wiki/spaces/gokart/pages/58294758/Knowledge+base+mgmt+in+new+UI+with+GIT+support

In the docs it's called category

Reply:
Yeah, I wrote 😛  it but I thought name would be better. Does category make more sense?

Reply:
To me name maybe might confusing people into think it's label-ey but I'm not too picky. As long as we keep it consistent and well documented.

Reply:
Just sayin.... I did have to ask you (Franz) if this could have spaces in it, mainly because I wasn't sure what this value represented

Reply:
Let's go with `category`

Reply:
Cool, I'll rename my shit and push the changes in just a moment

Reply:
Ok, so since it's the directory name, that means spaces aren't allowed?  Just A-Z, a-z, 0-9, - _  . 

Is that correct?

Reply:
Yup